tests/Backup-DbaDatabase.Tests.ps1

```diff
@@ -23,13 +23,22 @@ Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
         }
         Get-DbaDatabase -SqlInstance $script:instance1 -Database "dbatoolsci_singlerestore" | Remove-DbaDatabase -Confirm:$false
         Get-DbaDatabase -SqlInstance $script:instance2 -Database $DestDbRandom | Remove-DbaDatabase -Confirm:$false
+        $server = Connect-DbaInstance -SqlInstance $script:instance2
+        $sql = "CREATE CREDENTIAL [https://dbatools.blob.core.windows.net/sql] WITH IDENTITY = N'SHARED ACCESS SIGNATURE', SECRET = N'$env:azurepasswd'"
```

Comment:
here container is fixed....it'd be good to make it a variable . Also, if there are no details about a container and the relative credentials, test should be skipped as already done in the other test file

Reply:
this testdoes have `if ($env:azurepasswd) {` right above. good suggestion about making it flexible. I'd like to save that for a later PR 😴

Reply:
nope ... beforeall is not concerned about $env:azurepasswd

Reply:
oh good call, ty, iwll fix